.gitignore

```
@@ -0,0 +1,2 @@
+build/
+schemas/
```

The .xml files in schemas/ are hand authored, not generated, so you can't ignore the entire directory.
```suggestion
schemas/gschemas.compiled
```